### ISSUE TITLE
Crash when opening details of anime in Qt interface

### ISSUE THREAD

**Issue by veluria** [z411/trackma#222]
Traceback (most recent call last):
      File "/usr/lib/python3.5/site-packages/trackma/ui/qtui.py", line 1548, in r_details_loaded
        for line in details['extra']:
    KeyError: 'extra'

To reproduce: Open details page of Nejimaki Seirei Senki: Tenkyou no Alderamin.

**Comment by Birdulon** [comment]
That's a bad cache issue. Delete ~/.trackma/[user].[api]/anime.* and try running trackma again.

**Comment by zJoul** [comment]
The same occurs in gtk ui and cli (curse not tested) only with title containing specials caracters like " :,. " etc...

**Comment by Birdulon** [comment]
I can't reproduce with a fresh (v0.6) cache on qt, gtk or cli, checking _Nejimaki Seirei Senki: Tenkyou no Alderamin_ and _Time Travel Shoujo: Mari Waka to 8-nin no Kagakusha-tachi_ on Anilist. Curses appears to have some other problems at the moment which I'm currently debugging.

I was able to reproduce the error by checking the info on shows that I had checked previously on v0.4, before I cleared the relevant cache files. I strongly suspect the cause involves the difference in python3 string handling (perhaps keys are unicode strings now rather than bytestrings,) but the problem here seems to be that at least some of the old cache files aren't being automatically cleared with the version number change.
